src/renderer/helpers/playlists.js

```diff
@@ -45,6 +49,10 @@ function compareTwoPlaylistItems(a, b, sortOrder, collator) {
       return collator.compare(a.author, b.author)
     case SORT_BY_VALUES.AuthorDescending:
       return collator.compare(b.author, a.author)
+    case SORT_BY_VALUES.VideoLengthAscending:
+      return a.lengthSeconds - b.lengthSeconds
+    case SORT_BY_VALUES.VideoLengthDescending:
+      return b.lengthSeconds - a.lengthSeconds
```

Comment:
```suggestion
    case SORT_BY_VALUES.VideoLengthAscending: {
      const aLengthForSort = (isNaN(a.lengthSeconds) || a.lengthSeconds === 0) ? 0 : a.lengthSeconds
      const bLengthForSort = (isNaN(b.lengthSeconds) || b.lengthSeconds === 0) ? 0 : b.lengthSeconds
      return aLengthForSort - bLengthForSort
    }
    case SORT_BY_VALUES.VideoLengthDescending: {
      const aLengthForSort = (isNaN(a.lengthSeconds) || a.lengthSeconds === 0) ? 0 : a.lengthSeconds
      const bLengthForSort = (isNaN(b.lengthSeconds) || b.lengthSeconds === 0) ? 0 : b.lengthSeconds
      return aLengthForSort - bLengthForSort
    }
```

![image](https://github.com/user-attachments/assets/379bf273-026a-4048-9bcb-68a82e190776)
![image](https://github.com/user-attachments/assets/b8ff6697-d9f4-444e-a185-056c5e572ced)

Reply:
Commit 57f251d adds this in, I did catch that both of the cases had the same difference calculation, so I fixed that as well.